### PR TITLE
feat(sqllab): Add /sqllab endpoint to the v1 api

### PR DESCRIPTION
**Pull request by justinpark**
### SUMMARY
Part 1-b of SQLLab SPA migration: [detail link](https://docs.google.com/document/d/1opyNuaw-CEAD9tG6fx_zqNcZwe3_soQgsnSJTP4mQkk/edit?usp=sharing)

This commit introduces new api endpoint (`/api/v1/sqllab/`) and re-use the sqllab_bootstrap_data util to share the bootstrap data with the legacy landing page.

We will migrate the legacy bootstrap util by a command class after the SPA initiative is completed.

```
GET /v1/sqllab/

RESPONSE
{
  active_tab: <object>
  databases: <array of database object>
  queries: <hashmap of query object>
  tab_state_ids: <list of a tab id and name>
}
```
Screenshot 2023-08-14 at 2 08 25 PM

### TESTING INSTRUCTIONS
1 - Execute the API tests
2 - All tests should pass

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by EugeneTorap** [comment]
Thanks! Can you try to fix the tests?

File: tests/integration_tests/sql_lab/api_tests.py [hunk 1]
```diff
@@ -37,11 +37,76 @@
...
 
 
 class TestSqlLabApi(SupersetTestCase):
+    @pytest.mark.usefixtures("create_gamma_sqllab_no_data")
+    @mock.patch.dict(
+        "superset.extensions.feature_flag_manager._feature_flags",
+        {"SQLLAB_BACKEND_PERSISTENCE": False},
+        clear=True,
+    )
+    def test_get_from_bootstrap_data(self):
```
Comment (michael-s-molina) [inline tests/integration_tests/sql_lab/api_tests.py:52]: Is this trying to test a scenario where there's no data available? If so, could you improve the test name to something like `test_empty_bootstrap_data`?

File: tests/integration_tests/sql_lab/api_tests.py [hunk 2]
```diff
@@ -37,11 +37,76 @@
...
+        assert result["queries"] == {}
+        assert result["tab_state_ids"] == []
+        self.assertEqual(len(result["databases"]), 0)
+
+    @mock.patch.dict(
+        "superset.extensions.feature_flag_manager._feature_flags",
+        {"SQLLAB_BACKEND_PERSISTENCE": True},
+        clear=True,
+    )
+    def test_get_from_backend_persistence_payload(self):
```
Comment (michael-s-molina) [inline tests/integration_tests/sql_lab/api_tests.py:68]: Maybe rename to `test_bootstrap_data_queries`?

**Review by michael-s-molina — APPROVED** [review]
LGTM

**Pull request merged**